sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/ToProtoConverter.java

```diff
@@ -28,7 +28,9 @@ protected ToProtoConverter(final String tableName, Map<String, ToProtoValueCodec
     final ToProtoValueCodec codec = getCodec(fieldName);
     try {
       if (value instanceof String) {
-        return codec.protoValueFromStringified((String) value);
+        // Need to allow optional use of "extra" single quotes
+        String strValue = StringifiedValueUtil.handleSingleQuotes((String) value);
```

Comment:
Note: outermost handling here, to avoid codecs from all having to know about handling; "inner" single-quoting dealt with by "StringifiedValueUtil" for structured types (List, Set, Map, UDT, Tuple).